src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/PostgreSqlGQLSupportedTypesTests.cs

```diff
@@ -24,13 +24,38 @@ public static async Task SetupAsync(TestContext context)
             await InitializeTestFixture();
         }
 
-        protected override string MakeQueryOnTypeTable(List<string> columnsToQuery, int id)
+        /// <summary>
+        /// Postgres requires conversion of 0.33 to 'real' type otherwise precision is lost.
```

Comment:
This seems like a `Note` to me as opposed to a summary of this function. Why are we talking specifically about 0.33?

Reply:
good point, I think 0.33 is just an example, but not clear why the conversion was not needed for the last test

Reply:
I added the conversion for the last case as well.